app/bundles/InstallBundle/Configurator/Step/CheckStep.php

```diff
@@ -36,18 +33,22 @@ class CheckStep implements StepInterface
      */
     private $kernelRoot;
 
-    /** @var OpenSSLCipher */
+    /**
+     * @var OpenSSLCipher
+     */
     private $openSSLCipher;
 
     /**
      * Absolute path to cache directory.
+     * Required in step.
```

Comment:
What this comment means?

Reply:
Usage cannot be found in PHPStorm code inspection. This caused problems form me whith usages autocheck and is covered by some magic in the top of these steps.

That's why I leaved notice about this.

Reply:
Btw. I removed some attrs based on this check.